Using field for location, pagination on past exhibits, features

hllavina opened the pull request:
Using Location field instead of Category for location. this allows featured category to be used to place posts on home page. The display of current, past, upcoming has been changed to display location field rather than the category. The current, past, upcoming views have been condensed into one view, inc/exhibits-detail, the only difference is display of date

Commit:
Using field for location, pagination on past exhibits

Commit:
Place exhibits detail in correct inc place

Commit:
Update to display Location field instead  of category name

frrrances commented:
@hllavina can you get the code climate to pass? I'll take a spin through once those changes are up.

Commit:
Address tabs, code sniffer issues

Add link to Past exhibits from location pages; code sniffer

Update to resolve code sniffer issue

hllavina commented:
All the code climate issues  have been addressed except for the meta key and meta query issues which are out of scope for these updates. Libraries-dev have been updated with the new changes. thanks for your review

matt-bernhardt commented:
I've reviewed the remaining CodeClimate failures and approved them - I agree that it isn't necessary to tackle this issue at the moment.

frrrances commented on inc/exhibits-detail.php:
```diff
@@ -0,0 +1,59 @@
+				<div class="entry-categories">
+					<div class="entry-cats-list">
+
+							<a href="<?php echo esc_url( $cat_link ); ?>"><span class="category-bg"><span class="category-init"><?php echo esc_attr( substr( get_field( 'location' ),0,1 ) ); ?></span></span><span class="cat-name"><?php the_field( 'location' ) ?> Exhibit</span></a>
```
This link is using cat_link but now the rest is using location. Ideally, this would go to a listing of exhibits for that location, but on dev it is an empty href. Can you take a look and see what's possible to link to here?

hllavina replied:
Hi Thanks for the catch, I am fixing. I also noticed that the breadcrumbs are not correct , they should have the location name when displaying a particular location exhibit, should we address this now?

frrrances replied:
Good catch! Yes, that would be great.

matt-bernhardt reviewed (dismissed):
I believe this looks good. Provided it looks good to Frances, I'm comfortable merging this and proceeding into test or production as needed.

frrrances reviewed (commented):
One question on the location link, otherwise it looks and acts nicely. I'll let @matt-bernhardt weigh in on any of the WP loops and functions though. Once you address the link, let me know and I'll re-review.

hllavina committed:
Edit link to location

hllavina committed:
Update of breadcrumb reference

hllavina committed:
Update of breadcrumb reference

matt-bernhardt commented on inc/exhibits-detail.php:
```diff
@@ -0,0 +1,59 @@
+				<div class="entry-categories">
+					<div class="entry-cats-list">
+
+							<a href="/exhibits/<?php echo esc_attr( str_replace( ' ','-',get_field( 'location' ) ) ); ?>"><span class="category-bg"><span class="category-init"><?php echo esc_attr( substr( get_field( 'location' ),0,1 ) ); ?></span></span><span class="cat-name"><?php the_field( 'location' ) ?> Exhibit</span></a>
```
Is there a way to write this line so it isn't hard-coded to the Exhibits site? Is that available via a Wordpress function in some way?

matt-bernhardt reviewed (changes requested):
The only thing I'd like to see changed would be line 18 on the new detail partial - which looks like it is bound to the exhibits site right now? I believe there are plans to use Exhibits posts in more than just this site, so this seems like something that needs to be site-agnostic.

Everything else looks good.

hllavina committed:
Updating location page link
